spec/unit/queue_spec.rb

```diff
@@ -33,54 +33,54 @@
     it 'handles numbers' do
       expect(described_class.queue_name(step, -1)).to eq 'a_b_c_-1'
       expect(described_class.queue_name(step, 0)).to eq 'a_b_c_0'
-      expect(described_class.queue_name(step, 1)).to eq 'a_b_c_1'      
+      expect(described_class.queue_name(step, 1)).to eq 'a_b_c_1'
       expect(described_class.queue_name(step, 99)).to eq 'a_b_c_99'
     end
   end
-  
+
   context '#empty_slots' do
     before do
       Resque.redis = MockRedis.new
     end
-    
+
     it 'illegal arguments' do
       expect { described_class.empty_slots }.to raise_error(ArgumentError)
     end
-    
+
     it 'no queue' do
       described_class.empty_slots(step).should eq 100
     end
-    
+
     it 'queue single job' do
       Resque.enqueue_to(described_class.queue_name(step), 'Foo')
       described_class.empty_slots(step, :default, 1).should eq 0
     end
 
     it 'queue threshold jobs' do
-      threshold.times do |i|
+      threshold.times do |_i|
```

Comment:
This is an example where the rubocop auto-fixer would benefit from human eyes. Because the argument `_i` isn't used, we can just drop it entirely.